packages/editor/src/client/routes/commentList.tsx

```diff
@@ -183,6 +183,9 @@ export function CommentList() {
     })
   }
 
+  let printUsername = ` ${currentComment?.anonymousName} (anonymous)`
+  if (currentComment?.user) printUsername = `${currentComment?.user.name}`
```

Comment:
Translation for `anonymous`

Reply:
Maybe better to display as 'unregistered user' if there's a display name?

Reply:
For simplicity would a ternary assignment work better here?